### PR TITLE
Update order

### PR DESCRIPTION
**Pull request by avsek477**
**_Task_**: https://app.asana.com/0/1125859137800433/1201427008884285
**_Reference for order update payload_**: https://docs.bitfinex.com/reference#ws-auth-input-order-update

**_Changes required in frontend_**: Add translation in `bfx-hf-ui-core`
```
"updatingOrderWithId": "Updating order (ID: {{orderId}})"
```

**_Request payload_**: 
```
dispatch(WSActions.send(['order.update', authToken, {
      id: 80039464518,
      amount: '2.00000000',
      price: '33000',
      flags: 64,
      meta: { make_visible: 1 },
}]))
```
- NOTE: payload for stop limit orders: 
```
{
   id: 80039464526,
   price: '32000',
   price_aux_limit: '34000',
   amount: '4'
}
```

### CODE REVIEW

File: lib/ws_servers/api/handlers/on_order_update.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  } else if (!ws.clients.bitfinex) {
+    sendStatus('failed')
+    return sendError(ws, 'No client open for Bitfinex', ['noClientOpenFor'])
+  }
+
+  try {
+    notifyInfo(ws, `Updating order (ID: ${orderId})`, ['updatingOrderWithId', { orderId }])
+    await ws.clients.bitfinex.updateOrder(packet)
+
+    d(`sucessfully updated order [ID: ${orderId}]`)
```
Comment (tarcisiozf) [inline lib/ws_servers/api/handlers/on_order_update.js:38]:
```suggestion
    d(`successfully updated order [ID: ${orderId}]`)
```

typo

**Review by tarcisiozf — APPROVED** [review]
lgtm

File: lib/ws_servers/api/handlers/on_order_update.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    sendStatus('failed')
+    return
+  }
+
+  if (!isAuthorized(ws, authToken)) {
+    sendStatus('failed')
+    return sendError(ws, 'Unauthorized', ['unauthorized'])
+  } else if (!ws.clients.bitfinex) {
+    sendStatus('failed')
+    return sendError(ws, 'No client open for Bitfinex', ['noClientOpenFor'])
```
Comment (vitaliystoliarovcc) [inline lib/ws_servers/api/handlers/on_order_update.js:31]: i18n props are missing (`{ target: 'Bitfinex' }`)

**Review by vitaliystoliarovcc — CHANGES_REQUESTED** [review]

File: lib/ws_servers/api/handlers/on_order_update.js (hunk 2 above)
Comment (avsek477) [reply lib/ws_servers/api/handlers/on_order_update.js]: done

**Review by vitaliystoliarovcc — APPROVED** [review]

**Pull request merged**